extensions/blocks/podcast-player/view.js

```diff
@@ -0,0 +1,55 @@
+/* global _wpmejsSettings, MediaElementPlayer, jetpackPodcastPlayers */
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const playerInstances = {};
+const meJsSettings = typeof _wpmejsSettings !== undefined ? _wpmejsSettings : {};
+
+const initializeBlock = function( id ) {
+	const block = document.getElementById( id );
+	if ( block ) {
+		const player = {
+			id,
+			block,
+			audio: block.querySelector( 'audio' ),
+			currentTrack: 0,
+		};
+		playerInstances[ id ] = player;
+
+		// Initialize player UI.
+		player.mediaElement = new MediaElementPlayer( player.audio, meJsSettings );
+	}
+};
+
+// Initialze queued players.
+if ( window.jetpackPodcastPlayers !== undefined ) {
+	jetpackPodcastPlayers.forEach( initializeBlock );
+}
+
+// Replace the queue with an immediate initialization for async loaded players.
+window.jetpackPodcastPlayers = {
+	push: initializeBlock,
```

Comment:
I wonder if there is too much potential for this to break due if code ordering got messed up?